eclipse-theia/theia
# license: update baseline

#### What it does


The commit updates the `license-check-baseline`:
- adds `seek-bzip` with link to approved cq.
- updates `jsdom` and `jsmin` as they are now approved.
- removes extraneous entries.

#### How to test

- the `license` ci check should pass
- executing `yarn license:check` should produce no errors.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
